### PR TITLE
Added tests for RadioButton.

### PR DESCRIPTION
**Pull request by demogrommet**
Hi Grommet Team,

  I've added tests for the RadioButton.

### CODE REVIEW

File: src/js/components/RadioButton.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var RadioButton = React.createClass({
   render: function () {
     var classes = [CLASS_ROOT];
     if (this.props.className) {
-      classes.push(this.props.className);
+        classes.push(this.props.className);
```
Comment (demogrommet) [inline src/js/components/RadioButton.js]: Ups, sorry. I will fix that.

**Comment by demogrommet** [comment]
I believe I've fixed the issue! Can someone please review? Thanks!

File: test/components/RadioButton-test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  it('loads a basic RadioButton', function() {
+    var React = require('react/addons');
+    var Component = GrommetTestUtils.getComponent(__path__, undefined, {
+      id: 'radio-id',
+      label: 'Test Label'
+    });
+
+    GrommetTestUtils.componentShouldExist(Component, 'radio-button');
+    GrommetTestUtils.componentShouldExist(Component,
+      'radio-button__label', 'Test Label');
```
Comment (alansouzati) [inline test/components/RadioButton-test.js]: Great! I like that you are testing the label as well.

**Comment by alansouzati** [comment]
Thanks for your contribution!

**Pull request merged**